installer/install.iss

```diff
@@ -2692,6 +2699,32 @@ begin
         LogError(ExpandConstant('Line {#__LINE__}: Unable to remove the Git for Windows updater (output: '+ReadFileAsString(LogPath)+', errors: '+ReadFileAsString(ErrPath)+').'));
 end;
 
+procedure InstallWindowsTerminalFragment;
+var
+    Res:Longint;
+    AppPath,JSONPath:String;
+begin
+    if IsAdminInstallMode() then
+        JSONPath:=ExpandConstant('{commonappdata}\Microsoft\Windows Terminal\Fragments\Git\git-bash.json')
+    else
+        JSONPath:=ExpandConstant('{localappdata}\Microsoft\Windows Terminal\Fragments\Git\git-bash.json');
+    AppPath:=ExpandConstant('{app}');
+    StringChangeEx(AppPath, '\', '/', True)
+    if not SaveStringToFile(JSONPath,
+        '{'+
+        '    "profiles": ['+
+        '      {'+
+        '        "name": "Git Bash",'+
+        '        "commandline": "'+AppPath+'/usr/bin/bash.exe -i -l",'+
```

Comment:
I'm late to the party on this, but should this be `"'+AppPath+'/bin/bash.exe -i -l"`? That's what https://github.com/microsoft/terminal/issues/1394#issuecomment-740300541 says, referencing https://stackoverflow.com/questions/56839307/adding-git-bash-to-the-new-windows-terminal#comment110596050_57369284

Reply:
Excellent point. Would you kindly open a PR (I can't right now, am on a phone).

Reply:
PR created: #348

Reply:
And merged!